packages/editor-ui/src/components/SettingsSidebar.vue

```diff
@@ -111,6 +104,11 @@ export default mixins(
 </script>
 
 <style lang="scss" module>
+:global(.el-menu) {
```

Comment:
does this need to be global? could not we assign a class directly?

Reply:
These `element-ui` variable overrides have to sit it `.el-menu` class. The only other way is to move this in scoped or regural CSS block but I decided to keep everything in one place (<style> block) hence so many `:global` selectors.